include/OsmAndCore/ITypefaceFinder.h

```diff
@@ -25,11 +30,28 @@ namespace OsmAnd
     public:
         struct OSMAND_CORE_API Typeface Q_DECL_FINAL
         {
-            Typeface(const sk_sp<SkTypeface>& skTypeface, hb_face_t* hbTypeface);
+            Typeface(const sk_sp<SkTypeface>& skTypeface,
+                     THbFontPtr hbFont_,
```

Comment:
Can't see THbFontPtr definition - it should be just std::shared_ptr<hb_font>

Reply:
It was moved into separate file. File have been added.
`THbFontPtr` - just to use a short name.

Reply:
Again, it should be std::shared_ptr<hb_font> for reasons explained in https://github.com/osmandapp/OsmAnd-core/pull/279/files#r770836119

Reply:
It was moved into separate file. File have been added.
THbFontPtr - just to use a short name.